content/en/deployment/strategies/overview.md

```diff
@@ -8,106 +8,102 @@ categories: ["Deployment Strategies", "Features", "Concepts"]
+1. Once your requirements are fulfilled the "blue" environment is shut down and the leaving only the new version running.
 
-### Benefits of the Blue-Green Deployment Strategy
-The blue-green deployment strategy gives you the benefits of running tests against a full copy of your production environment 
-prior to any traffic being routed to the new version. This allows for zero-downtime deployments and reduced risk when exposing
-users to new versions of your application. 
+### Benefits of the blue/green deployment strategy
+The blue/green deployment strategy gives you the benefits of running tests against a full copy of your production environment prior to any traffic being routed to the new version. This gives you for zero downtime deployments and reduced risk when exposing users to new versions of your app. 
 
-In addition, because the blue-green strategy maintains the 'blue' environment until the shutdown conditions are met it 
-gives you the ability to instantly roll back to a known stable version should any issues be detected during the deployment. 
+In addition, because the blue/green strategy maintains the 'blue' environment until the shutdown conditions are met it, blue/green gives you the ability to instantly roll back to a known stable version should you detect any issues during the deployment. 
 
 ## When to use each strategy
 
-Both the canary and blue-green strategies have trade-offs that can help you determine which strategy is best for your use case. 
+Both the canary and blue/green strategies have trade-offs that can help you determine which strategy is best for your use case. 
 
-### When to use the Canary strategy
+### When to use the canary strategy
 
 Canary is best for use cases where you want to minimize risk by exposing your changes to a small percentage of users and 
 monitor prior to exposing the changes to the majority user base. This allows you to ensure stability and performance of 
 the new version before creating a widespread impact. This minimizes the impact of potential issues on user.
 
-### When to use the Blue-Green strategy
+### When to use the blue/green strategy
 
-Blue-Green deployments are great when you want the ability to rapidly rollback if issues are detected. Given that both 
+Blue/Green deployments are great when you want the ability to rapidly roll back if issues are detected. Given that both 
 the 'blue' and 'green' environments are running in parallel you have the ability to instantly shift traffic from one to
 the other. 
 
-In addition, the blue-green strategy can give you increased confidence by allowing to your test your changes in the 
+In addition, the blue/green strategy can give you increased confidence by allowing to your test your changes in the 
 'green' environment as a dark deployment before users are exposed to the changes minimizing any potential impacts. This 
 can be accomplished using the [CD-as-a-Service Preview Service]({{<ref "reference/deployment/config-preview-link">}})
 
```

Comment:
Extra blurb to consider adding -

Blue/Green deployment strategy is very simple to get started with and is recommended if you are just getting started with adopting advanced deployment strategies.